include/sys/euiAnimator.h

```diff
@@ -1,15 +1,16 @@
 #pragma once
 
 #include <nn/font/font_Util.h>
+#include <nn/ui2d/ui2d_Animation.h>
 #include <nn/ui2d/ui2d_Pane.h>
 
 namespace nn {
 namespace ui2d {
-    class Group;
-    class AnimTransformBasic;
-    class AnimResource;
-}
-}
+class Group;
+class AnimTransformBasic;
+class AnimResource;

```

Comment:
AnimTransformBasic and AnimResource can be removed seeing how they're defined in the required header (ui2d_Animation), Pane should be added to these forward decls imo

Reply:
Alright. I'll forward decl. Pane just because and remove AnimTransformBasic and AnimResource.